### DISCUSSION TITLE
How to setting faceRecognition?

### DISCUSSION THREAD

**Discussion by cwhzzang** [roflcoopter/viseron#316] [Q&A]
Please help me. I was setting face detecting for read explain. but i saw error. and don't operation.

**My Setting** 

cameras:
  - name: FrontYardAmcrest
    host: 192.168.0.50
    port: 554
    username: admin
    password: aqa79077907
    path: /cam/realmonitor?channel=1&subtype=0
    fps: 10
motion_detection:
  interval: 0.1
  trigger_detector: true
  timeout: true
  max_timeout: 5
  width: 300
  height: 300
  area: 0.1
  frames: 1
  threshold: 5
  alpha: 0.1

object_detection:
  type: darknet
  interval: 1
  labels:
    - label: person
      confidence: 0.5
    - label: cat
      confidence: 0.5

recorder:
  lookback: 5
  timeout: 5
  retain: 7
  folder: /recordings
  segments_folder: /segments
  extension: mp4
  thumbnail:
    save_to_disk: true
    send_to_mqtt: true

post_processors:
  face_recognition:
    type: dlib
    expire_after: 3
    unknown_faces_path: /home/cwhzzang/uncorrent_face
    face_recognition_path: /config/face_recognition
    model: cnn
  logging:
    level: debug

**My Forder**
My foder chmod is 777

/config/face_recognition/faces/person1/1.jpg
/config/face_recognition/faces/person1/2.jpg
/config/face_recognition/faces/person1/3.jpg

**- ERROR**

2022-01-29 22:17:19] [ERROR   ] [viseron.post_processors.face_recognition.dlib] - /config/face_recognition/faces does not exist. Make sure its created properly. See the documentation for the proper folder structure

Why don't operation? plase help me.

**Comment by roflcoopter** [comment]
Seems correct, however its hard to tell when the indentation of the config is missing.
Can you repaste your config between triple backticks (```)  so that indentation is not lost?
